blog/_posts/2021-05-18-argocd-private-clusters.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: post
+title: Bring GitOps to your private Kubernetes clusters with ArgoCD and inlets PRO
+description: Learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation and inlets PRO
+author: Johan Siebens
+tags: inlets-pro hybrid-cloud multi-cloud gitops argocd
+author_img: jsiebens
+image: /images/2021-05-18-argocd-private-clusters/background.jpg
+date: 2021-05-18
+---
+
+In this post, you'll learn how to apply GitOps on multiple private Kubernetes cluster with a single Argo CD installation.
+
+## GitOps with Argo CD
+
+Alexis Richardson coined the term GitOps in 2017. Since then, it’s gained significant interest in the enterprise companies because it allows for easier auditing of deployments.
```

Comment:
Link to Alexis on Twitter

Quote / italics GitOps

Do you have a link for this? I feel like we had very similar verbiage on another post recently. 

https://www.openfaas.com/blog/bring-gitops-to-your-openfaas-functions-with-argocd/

If it is copied from there, can you copy the markdown?